### DISCUSSION TITLE
Expand the support for footnotes, including via the import function

### DISCUSSION THREAD

**Discussion by volkantash** [GitbookIO/community#34] [Feature Requests]
Hi

How to make footnotes in gitbook?

This example is footnote of Github with markdown. 
![footnote](https://user-images.githubusercontent.com/25375593/214555201-10962d9a-b119-436c-9a3c-9297a8c124f5.gif)

**Comment by taszap** [comment]
Hey @volkantash, 

We have recently added support for footnotes in Markdown. They will be imported into GitBook as inline annotations. 

If you would like to create it directly in GitBook, select the word or phrase you would like to add the footnote to and click annotate! (alternatively you can use the following shortcut: ⌘⌥F) 

<img width="653" alt="image" src="https://user-images.githubusercontent.com/63643069/214872908-e3df8316-4d6a-426b-8969-3ac3c5c5b0da.png">

**Comment by volkantash** [comment]
Firstly. Annotate is not working in Firefox and Firefox Developer Edition. It closes as soon as it clicks.

Footnote is not your annotate. Why is the feature called annotate thought to be sufficient for footnote?

For example. I wrote a document with Libreoffice and created a footnote. I uploaded it to Gitbook. In what format will the article in Gitbook be presented?

**Reply by taszap** [reply]
Thanks for following up on this. Sorry to hear you had issues with Firefox and annotations - this has been reported to our engineering team for a fix. In the future I would recommend to ping support@gitbook.com because we don't monitor the community for bugs/fixes. 

Having said that, you are correct in saying that our support for footnotes is currently limited to Markdown - a footnote from Libreoffice would not be imported as an annotation but as ordered list at the bottom of your page. 

I will change the title of this feature request to make it clearer that you would like to see a support for footnotes (not as annotations) and the possibility to import them.

**Reply by maxim-ge** [reply]
**Hi @taszap,**  

As a [paid user](https://help.air.untill.com/), I want to emphasize the importance of proper footnote support in GitBook. Thank you for your response and for acknowledging the current limitations of GitBook’s footnote functionality. However, annotations are **not a suitable replacement** for footnotes. Footnotes should remain distinct elements, just as they are properly supported in GitHub using standard Markdown syntax (e.g., `[^1]`).  

## Feature Request: Proper Footnote Implementation

Could you clarify if there are plans to properly support footnotes in Markdown rendering? It would be ideal if GitBook could maintain footnotes in their correct format rather than converting them into inline annotations. Proper footnote support is essential for our structured documentation, and the current workaround does not meet the expected standards.

Looking forward to your response.  

Thank you!  
Maxim,
CTO
